tsfc/ufl_utils.py

```diff
@@ -21,6 +23,32 @@
                                      construct_modified_terminal)
 
 
+def compute_form_data(form,
```

Comment:
Ah, so this is here so that FFC does not need to duplicate the exact same options that TSFC works with.